src/pages/Welcome/index.tsx

```diff
@@ -0,0 +1,40 @@
+import { useNavigate } from 'react-router-dom';
+import {
+  Button, Logo, Subtitle, Title,
+} from '../../components/common';
+import { RouterPaths } from '../../constants';
+
+function Welcome() {
+  const navigate = useNavigate();
+
+  return (
+    <>
+      <div>
+        <Logo />
+        <Title text="Welcome to Target" />
+        <Subtitle text="Find people near you & connect" />
+      </div>
+      <div className="mb-1 text-left">
+        <ul>
+          <li className="mb-5 blue-dot">
+            Create a  target  by clicking wherever on the map, specify
+            the ratio and and a topic: Travel, Dating, Music, etc.
+          </li>
+          <li className="blue-dot">
+            Target will  start a chat whenever you&apos;ve a match.
+            You can always dismiss a conversation if you&apos;re not interested.
+          </li>
+        </ul>
+      </div>
+      <Button
+        type="button"
+        variant="primary"
+        text="Ok! Got it!"
+        maxWidth="max-w-xs"
+        onClick={() => navigate(RouterPaths.HOME)}
+      />
```

Comment:
Usually I would say that this is bad for accessibility since this should be an anchor with an `href`. I understand that maybe for this case since you want to take the user through a flow it is ok to use a button instead.

I just wanted to point this out to note that not because something looks like a button it means that it should use the `button` tag.
It also works the other way, not because something doesn't look like a button or a link it means that it can be a `div` with an `onClick`.